runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -1226,11 +1226,14 @@ ClientSessionHT::purgeOldDataIfNeeded()
             oldAge = OLD_AGE_UNDER_LOW_MEMORY; //memory is low
             }
          }
+
+      bool clientWasDeleted = false;
       // Time for a purge operation.
       // Scan the entire table and delete old elements that are not in use
       for (auto iter = _clientSessionMap.begin(); iter != _clientSessionMap.end(); ++iter)
          {
          TR_ASSERT(iter->second->getInUse() >= 0, "_inUse=%d must be positive\n", iter->second->getInUse());
+         clientWasDeleted = true;
```

Comment:
Just because we enter this loop, it doesn't mean that the clients were deleted. It only means that client sessions exist. Thus, I would change the name of the variable.

Reply:
True. I also changed the logic slightly, hopefully to be a little clearer.